### PR TITLE
LooksRareV2 support

### PR DESCRIPTION
**Pull request by hensha256**
Can finish once UR that supports LRV2 is published on NPM

### CODE REVIEW

File: test/forge/MixedSwapCallParameters.t.sol [hunk 1]
```diff
@@ -160,44 +163,21 @@ contract MixedSwapCallParameters is Test, Interop, DeployRouter {
...
+        assertEq(SEAPORT_NFT.balanceOf(RECIPIENT), 0);
         (bool success,) = address(router).call{value: params.value}(params.data);
         require(success, "call failed");
-        assertEq(balanceOfBefore - USDC.balanceOf(from), 58973906433);
-        assertEq(ethBalanceOfBefore - address(from).balance, 53000000000000000000);
-        assertEq(COVEN_NFT.balanceOf(RECIPIENT), 3);
-        assertEq(address(router).balance, 0);
-    }
-
-    function testMixedERC20ForNFTRevertsOnFailedUniswapSwap() public {
```
Comment (hensha256) [inline test/forge/MixedSwapCallParameters.t.sol:172]: This test was a perfect replica of the test above, so deleted it

**Review by zhongeric — APPROVED** [review]

File: src/entities/protocols/looksRareV2.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
+import abi from '../../../abis/LooksRareV2.json'
```
Comment (ewilz) [inline src/entities/protocols/looksRareV2.ts:1]: gittt, why the completely new file sans diff

**Review by ewilz — APPROVED** [review]
I approve with the caveat that I'm unfamiliar with LRV2 changes and assume you're confident with the changes you made to accompany it. all tests passing!

File: test/forge/utils/DeployRouter.sol [hunk 3]
```diff
@@ -59,16 +57,14 @@ contract DeployRouter is Test {
...
+                looksRareV2: LOOKS_RARE_V2,
                 routerRewardsDistributor: ROUTER_REWARDS_DISTRIBUTOR,
                 looksRareRewardsDistributor: LOOKSRARE_REWARDS_DISTRIBUTOR,
                 looksRareToken: LOOKS_TOKEN,
                 v2Factory: V2_FACTORY,
                 v3Factory: V3_FACTORY,
                 pairInitCodeHash: PAIR_INIT_CODE_HASH,
-                poolInitCodeHash: POOL_INIT_CODE_HASH,
-                paymentRecipient: PAYMENT_RECIPIENT,
-                paymentAmountBips: PAYMENT_BIPS
```
[the remark this answers is not in the document]
Comment (hensha256) [reply test/forge/utils/DeployRouter.sol:71]: This was a temporary change that we've actually never used on mainnet... this removal reverts it to how its always been

File: src/entities/protocols/looksRareV2.ts [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    merkleTree: MerkleTree
+  } {
+    const makerOrder: MakerOrder = { ...data.apiOrder }
+
+    const makerSignature: string = data.apiOrder.signature
+
+    const takerBid: TakerOrder = {
+      recipient: data.taker,
+      additionalParameters: '0x',
+    }
```
[the remark this answers is not in the document]
Comment (hensha256) [reply src/entities/protocols/looksRareV2.ts:118]: taker address of 0 sends the NFT to msg.sender - i.e. the router

**Pull request merged**